Clarify attributes on Message config topic (#8726)
magento/devdocs: src/guides/v2.4/extension-dev-guide/message-queues/config-mq.md

Parameter | Description
--- | ---
name | A string that uniquely identifies the topic. A topic name should be a series of strings that are separated by periods. The leftmost string should be the most general, and each string afterward should narrow the scope. For example, to describe actions for tending to pets, you might create names such as `cat.white.feed` and `dog.retriever.walk`. Wildcards are not supported in the `communication.xml` file.

> request | Specifies the data structure that’s used to define a message and is needed when publishing any messages through Magento.

Reviewer: Please avoid contractions
```suggestion
request | Specifies the data structure that is used to define a message and is needed when publishing any messages through Magento.
```

> request | Specifies the data structure that’s used to define a message and is needed when publishing any messages through Magento.

Reviewer 2: ```suggestion
request | Specifies the data structure that’s used to define a message. It is needed when publishing any messages through Magento.
```

response | Specifies the format of the response. This parameter is required if you are defining a synchronous topic. Omit this parameter if you are defining an asynchronous topic.

> schema | Describes the message structure. Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (Magento\Customer\Api\CustomerRepositoryInterface::save).

Reviewer 2: My comment on this line wasn't addressed. You don't mention what the Data Interface format is or differentiate it from `<module>\Api\<ServiceName>::<methodName>`.

> schema | Schema describes the message structure. Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (Magento\Customer\Api\CustomerRepositoryInterface::save).

Reviewer: Avoid conversational language
```suggestion
schema | Schema describes the message structure. Use the Data Interface format from the Service Contracts. There is an alternative way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (E.g. Magento\Customer\Api\CustomerRepositoryInterface::save).
```

> schema | Schema describes the message structure. Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (Magento\Customer\Api\CustomerRepositoryInterface::save).

Reviewer 2: ```suggestion
schema | Describes the message structure. Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:`<module>\Api\<ServiceName>::<methodName>` (Magento\Customer\Api\CustomerRepositoryInterface::save).
```

Reviewer 2: I don't understand "Use the Data Interface format from the Service Contracts. Besides, there is another way to specify a service method signature:"  Is `<module>\Api\<ServiceName>::<methodName>` not the preferred way to specify a schema name? You don't list the data interface format.

disabled | Determines whether this handler is disabled. The default value is `false`.

### `queue_consumer.xml` {#queueconsumerxml}

> As for consumer elements, they map the message receiver to a specific queue. Such parameters as class and method indicate what elements of the Magento 2 Message Queue Framework receive and process the message.

Reviewer: ```suggestion
With consumer elements, they map the message receiver to a specific queue. Such parameters as class and method indicate what elements of the Magento 2 Message Queue Framework receive and process the message.
```

> As for consumer elements, they map the message receiver to a specific queue. Such parameters as class and method indicate what elements of the Magento 2 Message Queue Framework receive and process the message.

Reviewer 2: ```suggestion
Consumer elements map the message receiver to a specific queue. Parameters such as class and method indicate which elements of the Message Queue Framework receive and process the message.
```

### `queue_publisher.xml` {#queuepublisherxml}

> The publisher element performs two vital actions: iit configures the type of connection and the same time sets up the exchange to publish to. Magento 2 uses only one exchange out of the box, but multiple exchanges are possible. As for the name of every exchange, it is a part of publisher configuration.

Reviewer: ```suggestion
The publisher element performs two vital actions, it configures the type of connection and at the same time, sets up the exchange to publish to. Magento uses only one exchange out of the box, but multiple exchanges are possible. The name for every exchange is contained in the publisher configuration.
```

> The publisher element performs two vital actions: iit configures the type of connection and the same time sets up the exchange to publish to. Magento 2 uses only one exchange out of the box, but multiple exchanges are possible. As for the name of every exchange, it is a part of publisher configuration.

Reviewer 2: ```suggestion
The publisher element performs two vital actions: 

*  It configures the type of connection.
*  It sets up the exchange to publish to. 

Magento uses only one exchange out of the box, but multiple exchanges are possible. The name of every exchange is a part of the publisher configuration.
```

```

#### `publisher` element

> {:.no_toc}  

Reviewer 2: Delete the added spaces at the end of this line
